web/download.rst

```diff
@@ -80,3 +80,19 @@ and then to build it::
 
 You should then add the ``./bin`` (make sure to expand this into an
 absolute path) directory to your ``PATH``.
+
+
+Docker Hub
+----------
+
+The `official Docker images <https://hub.docker.com/r/nimlang/nim/>`_
+are published Docker Hub and include the compiler and Nimble. There're images
```

Comment:
@Araq No, it's just a contraction of "there are." But I'll fix it anyway, just in case.